src/edu/washington/escience/myria/MyriaSystemConfigKeys.java

```diff
@@ -57,98 +56,76 @@ private MyriaSystemConfigKeys() {
    * */
   public static final String TCP_CONNECTION_TIMEOUT_MILLIS = "tcp.connection.timeout.milliseconds";
 
-  /**
-   * .
-   * */
-  public static final String WORKER_STORAGE_DATABASE_SYSTEM = "worker.storage.database.type";
+  /** */
+  public static final String WORKER_STORAGE_DATABASE_SYSTEM = "dbms";
 
-  /**
-   * .
-   * */
-  public static final String WORKER_STORAGE_DATABASE_CONN_INFO = "worker.storage.database.conn.info";
+  /** */
+  public static final String WORKER_STORAGE_DATABASE_NAME = "database_name";
 
-  /**
-   * .
-   * */
-  public static final String WORKER_IDENTIFIER = "worker.identifier";
+  /** */
+  public static final String WORKER_STORAGE_DATABASE_PASSWORD = "database_password";
 
   /** */
-  public static final String WORKING_DIRECTORY = "working.directory";
+  public static final String WORKER_STORAGE_DATABASE_PORT = "database_port";
+
+  /** */
+  public static final String WORKER_IDENTIFIER = "worker_id";
   /** */
-  public static final String DESCRIPTION = "description";
+  public static final String DEPLOYMENT_PATH = "path";
+  /** */
+  public static final String DESCRIPTION = "name";
   /** */
   public static final String USERNAME = "username";
   /** */
-  public static final String MAX_HEAP_SIZE = "max.heap.size";
+  public static final String MAX_HEAP_SIZE = "max_heap_size";
+  /** */
+  public static final String DEPLOYMENT_CONF_FILE = "deployment.cfg";
   /** */
-  public static final String DEPLOYMENT_FILE = "deployment.file";
+  public static final String ADMIN_PASSWORD = "admin_password";
   /** */
-  public static final String ADMIN_PASSWORD = "admin.password";
+  public static final String REST_PORT = "rest_port";
+  /** */
+  public static final String SSL = "ssl";
+  /** */
+  public static final String DEBUG = "DEBUG";
 
   /**
-   * Add default configurations into a configuraion.
+   * Add default configurations into a configuration.
    * 
    * @param config the configuration.
    * */
-  public static void addDefaultConfigKeys(final Map<String, String> config) {
-    if (!config.containsKey(FLOW_CONTROL_WRITE_BUFFER_HIGH_MARK_BYTES)
-        || config.get(FLOW_CONTROL_WRITE_BUFFER_HIGH_MARK_BYTES) == null) {
-      config.put(FLOW_CONTROL_WRITE_BUFFER_HIGH_MARK_BYTES,
+  public static void addDefaultConfigValues(final MyriaConfiguration config) {
```

Comment:
Could we factor out all this default-setting into a helper? E.g., `setDefaultValueIfMissing(String section, String key, String defaultValue)`